mlrun/api/utils/builder.py

```diff
@@ -358,11 +358,10 @@ def build_image(
 
     context = "/context"
     to_mount = False
-    v3io = (
-        source.startswith("v3io://") or source.startswith("v3ios://")
-        if source
-        else None
-    )
+    v3io = False
```

Comment:
v3io is to broad, with boolean params I suggest using `is_` prefix e.g `is_v3io_source`